api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -168,6 +168,11 @@ message ExternalProcessor {
   // :ref:`override_message_timeout <envoy_v3_api_field_service.ext_proc.v3.ProcessingResponse.override_message_timeout>`
   // If not specified, by default it is 0, which will effectively disable the ``override_message_timeout`` API.
   google.protobuf.Duration max_message_timeout = 10;
+
+  // Prevents clearing the route-cache when the
+  // :ref:`clear_route_cache <envoy_v3_api_field_service.ext_proc.v3.CommonResponse.clear_route_cache>`
+  // field is set in an external processor response.
+  bool disable_route_cache_clearing = 11;
```

Comment:
Please add a reference to this field in the comment of `CommonResponse.clear_route_cache`.
Also, WDYT about renaming the field to `disable_clear_route_cache`? I don't feel strongly about the name, but I think the motivation is when searching for a name in the docs, it should be able to return results for the two fields.